crates/nu-cli/src/shell/filesystem_shell.rs

```diff
@@ -995,10 +994,15 @@ impl Shell for FilesystemShell {
 
         let mut all_targets: HashMap<PathBuf, Tag> = HashMap::new();
         for target in targets {
-            if target.item.to_str() == Some(".") || target.item.to_str() == Some("..") {
+            let all_dots = target
+                .item
+                .to_str()
+                .map_or(false, |v| v.chars().all(|c| c == '.'));
```

Comment:
this is prepping for allowing an arbitrary number of dots to represent arbitrary ancestor directories.